test/common/statHelpers.test.js

```diff
@@ -0,0 +1,58 @@
+var sinon = require('sinon');
+var chai = require("chai")
+chai.use(require("sinon-chai"))
+var expect = chai.expect
+
+var shared = require('../../common/script/index.js');
+
+describe('helper functions used in stat calculations', function() {
+
+  var HEALTH_CAP = 50;
+  var LEVEL_CAP = 100;
+  var LEVEL = 57;
+  var BONUS = 600;
+  var MAXIMUM = 200;
+  var HALFWAY = 75;
+
+  it('provides a maximum Health value', function() {
+    expect(shared.maxHealth).to.eql(HEALTH_CAP);
```

Comment:
Health cap is only used here. I'd declare it in this it block since it doesn't get used in the rest of the tests.